locksmith/src/controllers/purchaseController.ts

```diff
@@ -22,13 +22,7 @@ namespace PurchaseController {
     if (!(await authorizedLock(lock))) {
       return res.sendStatus(451)
     }
-    const paymentProcessor = new PaymentProcessor(
```

Comment:
Is this going to change how _existing_ purchases work? We should make sure that the current flow is not broken.

Reply:
Ha I seee it does not change things ;)